consensus/system_contract/system_contract.go

```diff
@@ -0,0 +1,174 @@
+package system_contract
+
+import (
+	"context"
+	"fmt"
+	"github.com/scroll-tech/go-ethereum"
+	"github.com/scroll-tech/go-ethereum/accounts/abi"
+	"math/big"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/scroll-tech/go-ethereum/common"
+	"github.com/scroll-tech/go-ethereum/log"
+	"github.com/scroll-tech/go-ethereum/params"
+	"github.com/scroll-tech/go-ethereum/rollup/sync_service"
+)
+
+const (
+	defaultSyncInterval = 10
```

Comment:
```suggestion
	defaultSyncInterval = 10 * time.Second
```

if you use it like this in `syncTicker := time.NewTicker(defaultSyncInterval)` this means it is `time.Nanosecond`

Reply:
Thanks. Rejecting suggestion but committing this with other suggestions at once.